http-endpoint-example/README.md

```diff
@@ -0,0 +1,46 @@
+# How to download Met Office atmospheric model data using an HTTP endpoint
+A demonstration code example of downloading objects in response to SNS messages via an HTTP endpoint.
+
+By using the following you will be able to subscribe to an Amazon Web Services (AWS) Simple Notification Service (SNS) topic and start downloading objects.
+The running python script will accept notifications from the SNS topic and download the objects according to the input parameters.
+
+
+## Prerequisites:
+
+  * A service API key
+  * A Python 3 environment containing the "flask" and "requests" libraries
+  * A server with a Public IP or DNS address with an open port. Ports other than standard http/https are not reconmmended as described [here](https://aws.amazon.com/sns/faqs/ "AWS documentation")
+  * If used in production a method of verifying the messages are from AWS, this can be found [here](http://docs.aws.amazon.com/sns/latest/dg/SendMessageToHttp.verify.signature.html "AWS documentation")
+
+# Actions:
+1. ### Clone repo onto the server
+
+2. ### Start listening for notifications
+   Start polling the SNS topic by running host.py with the following command
+````
+  python host.py <port> <api_key> <start_time> <end_time> <comma-separated diagnostic names)
+````
+  * The port argument is the port you want the service to listen on
+  * Note that the start and end times are the forecast times (i.e. T+ times), in hours e.g. 0 to 24, up to a maximum of 144 hours (6 days).
+    * The start_time is the start of the forecast period
+    * The end_time is the end of the forecast period
+  * The 'temperature' argument returns the diagnostic 'surface_temperature',
```

Comment:
Could be worth putting these in the context of the argument name - 'comma-separated diagnostic names'. It may not be perfectly clear on first read